doc/langref.html.in

```diff
@@ -11631,7 +11650,7 @@ fn readU32Be() u32 {}
       {#header_close#}
 
       {#header_open|Grammar#}
-      <pre><code>Root &lt;- skip container_doc_comment? ContainerMembers eof
+      {#syntax_block|peg|zig.peg#}Root &lt;- skip container_doc_comment? ContainerMembers eof
```

Comment:
Since the grammar is pre-HTML escaped, using `syntax_block` here double-escapes it, causing HTML escape sequences such as `&lt;` to appear on the page instead of `<`.

Reply:
If you do choose to use the `syntax_block` here, the file's name is `grammar.y`. It is located at https://github.com/ziglang/zig-spec/blob/master/grammar/grammar.y.

Reply:
For style, I recommend using the same background color as zig's `figcaption`:
```
    figcaption.peg-cap {
      background: #fcdba5;
    }
```
Search for `figcaption.zig-cap`

Reply:
I think it would be nice to use `syntax_block` here, being able to directly copy paste the grammar from the `zig-spec` repo without needing to deal with the escaping would make keeping them in sync more pleasant.